README.md

```diff
@@ -38,13 +38,13 @@ A community made GraphQL API for Escape from Tarkov
 - Detailed info on medicines, stims, and in-game healing
 - So much more (it would take up this entire page to list everything 😸)
 
-> This [API](https://api.tarkov.dev/___graphql) does almost everything you would ever want for EFT!
```

Comment:
In this instance, the /graphql is actually correct because it's referring to the API url and not the playground url. I would make the changes myself, but I'm on mobile and not sure how to make that happen

Reply:
I guess it used the playground url before, so it probably doesn't matter.